HackYourFuture-CPH/simply-name.it
# delete board modal created

# Description

Modal view to delete the board

Fixes #88

# How to test?
#Check results without functionality
localhost:3000/deleteBoard
Delete the board and an alert will pop up saying its deleted.


#To check the results from owners page (with all functionality)
1.  const [modalVisibility, setModalVisibility] = useState(false);
2. <li onClick={() => {setModalVisibility(true);}}> Delete Board</li>
3.       {modalVisibility && (
        <DeleteBoardModal
          boardInfo={boardInfo}
          modalVisibility={modalVisibility}
          setModalVisibility={setModalVisibility}
        />
      )}
in board page when the delete button is clicked, the modal will pop up and the board can be deleted . Can we checked in the database for the deleted board along with the cascading deletions to members, results, candidates, ballots


# Checklist
